tests/Frameworks/Laravel/Version_5_7/composer.json

```diff
@@ -16,7 +16,7 @@
         "fzaninotto/faker": "^1.4",
         "mockery/mockery": "^1.0",
         "nunomaduro/collision": "^2.0",
-        "phpunit/phpunit": "^7.0"
+        "phpunit/phpunit": "<10"
```

Comment:
This is required because when `tests/save_code_coverage.php` calls to `(new PhpReport())->process` there is a misalignment of versions between the framework php unit and the one we use to save the report. This results on a fatal error